contracts/test/integration/IntegrationTestPriceBoundPSMUSDC.t.sol

```diff
@@ -115,8 +116,19 @@ contract IntegrationTestPriceBoundPSMUSDCTest is DSTest {
 
     /// @notice PSM is set up correctly and redeem view function is working
     function testGetRedeemAmountOut() public {
-        uint256 amountVoltIn = 100e12;
-        assertEq(psm.getRedeemAmountOut(amountVoltIn), 101);
+        uint256 amountVoltIn = 100e18;
+
+        uint256 currentPegPrice = oracle.getCurrentOraclePrice().div(1e12);
+        uint256 fee = (amountVoltIn * psm.redeemFeeBasisPoints()) /
+            Constants.BASIS_POINTS_GRANULARITY;
+
+        uint256 amountOut = ((amountVoltIn * currentPegPrice).div(1e18)) - fee;
+
+        assertApproxEq(
+            psm.getRedeemAmountOut(amountVoltIn).toInt256(),
+            amountOut.toInt256(),
+            1
```

Comment:
theoretically the math we did here should match up exactly with the numbers the PSM arrives at, so basis point deviation should be 0, so we should be able to use assertEq